extensions/flac/src/main/jni/flac_parser.cc

```diff
@@ -438,9 +438,11 @@ size_t FLACParser::readBuffer(void *output, size_t output_size) {
   return bufferSize;
 }
 
-int64_t FLACParser::getSeekPosition(int64_t timeUs) {
+int64_t* FLACParser::getSeekPosition(int64_t timeUs) {
```

Comment:
If this is now returning up to two seek positions, it should be called `getSeekPositions` or `getSeekPoints` (probably better if it's going to return a `SeekPoints` object. Similar naming comments apply elsewhere.

Reply:
Because there is no SeekPoints in C/C++ code, to do so I have to create a new class in C/C++. So, I just made it "getSeekPositions" simply.